packages/uxpin-merge-cli/src/program/args/ProgramArgs.ts

```diff
@@ -11,6 +11,7 @@ export interface RawProgramArgs {
 }
 
 export interface ConfigEnabledProgramArgs {
+  pageHeadContent?: string;
```

Comment:
What do you think about changing the type to string[] ?
it would be more generic I think 🤔